src/services/GraphApplication.ts

```diff
@@ -7,7 +7,7 @@ import { CreateDashboardGraph } from '../subgraphs/createDashboard';
 
 type SubgraphConfig = {
   name: string;
-  Graph: new (databaseId: number, clientAgentFunctions: Function[], companyName: string) => any;
+  Graph: new (databaseId: number, clientAgentFunctions: Function[], companyName: string, outputHandler: Function) => any;
```

Comment:
clientAgentFunctions: Function[] used to be the output handler, why are we no longer using it?
I'm ok just using the new variable, but should use only one